intelliserver/api/functions/data/index.js

```diff
@@ -0,0 +1,105 @@
+const express = require('express');
+const path = require('path');
+const axios = require('axios');
+const router = express.Router();
+const { ChatGPTInput, LLamaReplicateInput, LLamaSageInput, CohereInput } = require('intellinode');
+const { Chatbot, SupportedChatModels } = require('intellinode');
+const { USE_DEFAULT_KEYS } = require(path.join(global.__basedir, 'config'));
+
+const keys = {
+    'openai': process.env.OPENAI_API_KEY,
+    'replicate': process.env.REPLICATE_API_KEY,
+    'sagemaker': process.env.SAGEMAKER_API_KEY,
+    'cohere': process.env.COHERE_API_KEY
+};
+
+async function callSemanticSearchAPI(data) {
```

Comment:
add swager documentation. you can use openai to auto generate the documentation.

check example under: function/chatbot/index.js

Reply:
feedback implemented

Reply:
feedback implemented